app/cdap/components/ColumnInsights/Components/InputSelect/__test__/InputSelect.test.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import { fireEvent, render } from '@testing-library/react';
+import { createBrowserHistory as createHistory } from 'history';
+import React from 'react';
+import { Route, Router, Switch } from 'react-router';
+import InputSelect from '..';
+import { CHAR_ENCODING_OPTIONS } from '../../ParsingPopupBody/options';
```

Comment:
Add relative path to import starting from components/ and also format imports

Reply:
Did not review the test files.